src/algorithm/hnswlib/BlockManager.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright 2024-present the vsag project
```

Comment:
code style: add a blank line before this line

Reply:
done